nuget.config

```diff
@@ -1,5 +1,14 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
+  <config>
+    <!--
+      Where packages should go.  Visual Studio will provide a default value of $(SolutionDir) but command-line based
+      package restores could end up in a different location.  Setting this value keeps Visual Studio and NuGet.exe
+      in sync for packages.config based restores.
+    -->
+    <add key="globalPackagesFolder" value=".packages" />
+    <add key="repositoryPath" value=".packages" />
```

Comment:
what's the difference between these two?

Reply:
The globalPackagesFolder is a global cache of NuGet packages. This is where NuGet downloads NuGet packages to as a machine cache. The repositoryPath is the packages directory a solution will use, when using a packages.config file. Generally they are kept same. Combination of both of these helped to get rid of the signing problem I was facing where the 3rd party nuget packages were not getting signed properly, because they were getting downloaded at the default "{username}/.nuget/packages" location.